eclass/verify-sig.eclass

```diff
@@ -119,7 +119,7 @@ verify-sig_src_unpack() {
 		# find all distfiles and signatures, and combine them
 		for f in ${A}; do
 			found=
-			for suffix in .sig; do
+			for suffix in {.asc,.sig}; do
```

Comment:
```suggestion
			for suffix in .asc .sig; do
```

Reply:
Also these are PGP signatures.

Also, please ask upstream to host the file somewhere because keyserver are unreliable and blah blah blah.